ui/app/src/components/ContentTypeManagement/descriptors/controls/index.ts

```diff
@@ -376,22 +384,14 @@ export const sectionDescriptor: PartialContentType = {
 		expandByDefault: {
 			id: 'expandByDefault',
 			type: 'checkbox',
-			name: 'Expand by default',
-			description: 'Check this to show the section expanded when the content type is displayed in the content form',
+			name: defineMessage({ defaultMessage: 'Expand by default' }),
+			description: defineMessage({
+				defaultMessage: 'Check this to show the section expanded when the content type is displayed in the content form'
+			}),
 			helpText: '',
 			defaultValue: undefined,
 			validations: immutableEmptyObject
 		}
-		// TODO: create control for managing fields(?)
```

Comment:
What happened to this?

Reply:
That was done in the sections of the type. I'll add to the follow up ticket to check